server/dex/dex.go

```diff
@@ -113,6 +116,42 @@ type DEX struct {
 	bookRouter  *market.BookRouter
 	stopWaiters []subsystem
 	server      *comms.Server
+	config      *configResponse
+}
+
+// configResponse is defined here to leave open the possibility for hot
+// adjustable parameters while storing a pre-encoded config response message. An
+// update method will need to be defined in the future for this purpose.
+type configResponse struct {
+	configMsg *msgjson.ConfigResult // constant for now
+	configEnc json.RawMessage
+}
+
+func newConfigResponse(cfg *DexConf, cfgAssets []msgjson.Asset, cfgMarkets []msgjson.Market) (*configResponse, error) {
+	configMsg := &msgjson.ConfigResult{
+		BroadcastTimeout: uint64(cfg.BroadcastTimeout.Seconds()),
+		CancelMax:        cfg.CancelThreshold,
+		RegFeeConfirms:   uint16(cfg.RegFeeConfirms),
```

Comment:
We should consider making all the other confirm fields `uint64`.  We tend to think of smaller confirm counts with blocks in minutes, but others are much much faster, ETH for example.  Further, I expect that the notion of confirms could be used differently by different assets.

Reply:
> Further, I expect that the notion of confirms could be used differently by different assets.

If it differs too much, then our validation routines are not valid.

Reply:
My statement was overly vague.  Certainly the >= comparison needs to be meaningful regardless.  I'm imagining assets where:
1. blocks might be very fast
2. confirmation requirements might be exceptionally high for unknown reasons
3. confirmation might be more than a block count, perhaps a more direct reflection of work done since mining of the txn.

Reply:
Anyway, for registration fees, we're talking about DCR so these things aren't the case, but it reminded me of the other uint16 confirmation fields.